JqueryExercise/movies.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Movies</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+        integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/all.css">
+</head>
+
+<body>
+    <div class="main-heading text-center" id="Title">
+        <h1>Favorite Movies</h1>
+
+        <form class="form-inline justify-content-center" id="form">
+            <div class="Input">
```

Comment:
Technically,  there's nothing wrong with capitalizing classes, but I'd avoid it, especially not mixing with lowercase, used elsewhere.  Html classes are case sensitive.